.github/workflows/test-quaint.yml

```diff
@@ -16,14 +17,6 @@ jobs:
       matrix:
         features:
           - "--lib --features=all"
-          - "--lib --no-default-features --features=sqlite"
-          - "--lib --no-default-features --features=sqlite --features=pooled"
-          - "--lib --no-default-features --features=postgresql"
-          - "--lib --no-default-features --features=postgresql --features=pooled"
-          - "--lib --no-default-features --features=mysql"
-          - "--lib --no-default-features --features=mysql --features=pooled"
-          - "--lib --no-default-features --features=mssql"
-          - "--lib --no-default-features --features=mssql --features=pooled"          
```

Comment:
I'm not sure about this change. If we don't want to test that the crate works or even compiles without all features, should we just get rid of all these features since quaint is not a standalone library anymore? Or is the idea that we want to keep them if we want to build separate versions of QE per connector in the future, and those will be tested end-to-end instead?

Reply:
> Or is the idea that we want to keep them if we want to build separate versions of QE per connector in the future, and those will be tested end-to-end instead?

That's one of the ideas, yes. On the one hand, we wanted to have the possibility to split by connector, on the other hand it could be argued that we can remove all flags, and recover them from the git history if needed (YAGNI for the moment), but  removing the flags won't provide any user value or big maintainability win given how often do we change quaint today.

The combination of the above two things and the fact that we don't publish quaint any more makes testing individual flags right now unnecessary IMO.